packages/neptune-extension/src/common/api/backend.js

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios';
+
+export function authorize() {
+  const apiToken = window.localStorage.getItem('neptune_api_token');
+  const decodedToken = JSON.parse(atob(apiToken));
+
+  axios.defaults.baseURL = decodedToken.api_address;
+
+  return axios.get('/api/backend/v1/authorization/oauth-token', {
```

Comment:
`/api/backend/v1` is a common part of the URI, which can be eventually changed in the future.
We can parametrize it.